charts/k6-operator/templates/_helpers.tpl

```diff
@@ -89,7 +89,7 @@ Create the name of the service account to use
 {{- end -}}
 
 {{- define "k6-operator.namespace" -}}
-  {{- if eq .Release.Namespace "default" }}
+  {{- if eq .Release.Namespace "monitoring" }}
```

Comment:
Looks to me that this function is designed to default the installation into `{{ .Release.Name }}-system` instead of Helms default choice of the `default` namespace, in the scenario that the user doesn't specify any namespace using `helm install --namespace`.

This change would mean that if a user specifies `monitoring` as the desired namespace, the installation goes into `{{ .Release.Name }}-system`. In contrast, if a user doesn't specify any namespace, the installation goes into `default` (which is what this function is designed to prevent).